docs/install.rst

```diff
@@ -64,7 +64,7 @@ To run all tests in all environments, simply run ``tox``
 You may not have all Python versions installed or
 you may want to run a specific one.
 Have a look at the `documentation
-<https://tox.wiki/en/latest/example/general.html#selecting-one-or-more-environments-to-run-tests-against>`__.
+<https://tox.wiki/en/4.16.0/user_guide.html#cli>`__.
```

Comment:
This should be the stable version so that it is always current, instead of an outdated release. Also this is not an anonymous link, so only one underscore is needed.

```suggestion
<https://tox.wiki/en/stable/user_guide.html#cli>`_.
```